index.js

```diff
@@ -141,6 +142,11 @@ function isbundle(zipfile, callback) {
       return callback(null, true);
     }
 
+    //bundle: converted from gpx/kml (1 to n geojsons) and has metadata.json file and archived gpx/kml file
+    if (has_archivedfile && has_metadata && unique_extensions.indexOf('.geojson') > -1) {
```

Comment:
Will valid bundles with valid archived original ever get here? Seeing a the [check above](https://github.com/mapbox/bundle-fairy/pull/9/files#diff-168726dbe96b3ce427e7fedce31bb0bcR141) just checks for `has_metadata` and `.geojson` extensions. A bundle with an archived file will evaluate to true there, and return.